internal/receiver/smartagentreceiver/config_test.go

```diff
@@ -342,19 +343,35 @@ func TestLoadInvalidConfigWithInvalidEndpoint(t *testing.T) {
 	require.Nil(t, cfg)
 }
 
-func TestLoadInvalidConfigWithUnsupportedEndpoint(t *testing.T) {
+// Even though this smart-agent monitor does not accept endpoints,
+// we can create it without setting Host/Port fields.
+func TestLoadConfigMonitorNoHostPort(t *testing.T) {
```

Comment:
nit, since this test is about the endpoint value I'd maybe update to something like `TestLoadConfigWithUnsupportedEndpoint `